src/components/Face/index.js

```diff
@@ -18,8 +18,8 @@ export const FaceOverlay = () => {
   )
 }
 
-export const FaceInstructions = ({ handeClick }) => {
-  return (
+export const FaceInstructions = ({method, handeClick }) => {
+  if (method === 'face') return (
```

Comment:
why is this checking the method, should it not be called only if it is indeed the face step?